lib/src/components/NxPageHeader/__tests__/NxPageHeader.test.tsx

```diff
@@ -5,84 +5,119 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { shallow } from 'enzyme';
-import 'jest-enzyme';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
 import NxPageHeader from '../NxPageHeader';
-import AbstractNxPageHeader from '../../AbstractNxPageHeader/AbstractNxPageHeader';
 
-describe('NxPageHeader', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxPageHeader, {});
+import { screen } from '@testing-library/react';
+import { rtlRenderElement } from '../../../__testutils__/rtlUtils';
+
+fdescribe('NxPageHeader', function() {
```

Comment:
```suggestion
describe('NxPageHeader', function() {
```

Reply:
Jest has ways to filter which tests are running on the fly, so you don't generally need to do this.  For instance, in `yarn test-watch`, it has menu options allowing you to filter by filename or test name.  On the command line for `yarn jest`, you can also pass the filename as an arg or use the `--testNamePattern` option.